content/stratis-release-notes-3.8.1.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "Stratis 3.8.1 Release Notes"
+date = 2025-05-13
+weight = 41
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+Stratis 3.8.1, which consists of `stratisd 3.8.1` and `stratis-cli 3.8.1`
+is a bug fix release which addresses the following problems.
+
+For `stratis-cli`, the release fixes a bug where a user would be unable to
+start an encrypted pool previously created with any Stratis release less
+than 3.8.0.
+
+`stratis-cli 3.8.1` also contains better organized help text and displays 
+more complete information in the pool detail view.
+
+For `stratisd`, the release makes improvements to the Stratis support for
+mounting filesystems at boot. It introduces a new systemd unit file,
+`stratis-fstab-setup-with-network@.service`, which should be used when a
+filesystem's pool requires unlocking with the network present, as is the
+case when a pool is encrypted using NBDE (network-bound disk encryption).
+The fstab entry for the filesystem must include the `_netdev` option if this
+systemd unit file is used.
+
+An example fstab entry for a filesystem on a pool that is encrypted using
+NBDE should look something like this:
+
+`/dev/stratis/<POOL_NAME>/<FILESYSTEM_NAME> <MOUNTPOINT> xfs defaults,_netdev,x-systemd.requires=stratis-fstab-setup-with-network@<POOL_UUID>.service,x-systemd.after=stratis-fstab-setup-with-network@<POOL_UUID>.service 0 2`
+
+The `_netdev` option ensures that systemd delays attempting to mount the
```

Comment:
This is not exactly what it does. It avoids a cyclical dependency in systemd.

Reply:
I added an additional sentence which hopefully will clarify. How does that cycle manifest in systemd operation?

Reply:
The cycle will cause boot to fail in really spectacularly unpredictable ways such as the graphical interface not coming up but you still should be able to switch to a terminal in the console to log in and repair it in my experience. The logs will always clarify that there is a cyclical dependency.